src/realm/util/interprocess_mutex.hpp

```diff
@@ -112,9 +122,11 @@ class InterprocessMutex {
     friend class InterprocessCondVar;
 };
 
-
 inline InterprocessMutex::InterprocessMutex()
 {
+#ifdef REALM_ROBUST_MUTEX_EMULATION
+    std::call_once(static_flag, static_initialisation);
```

Comment:
But on the other side, using call_once makes it more explicit what happens, which may be a good thing.